src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py

```diff
@@ -9,73 +9,76 @@
 
 helps['storage entity insert'] = """
     type: command
-    short-summary: Insert an entity into the table.
-    long-summary: Inserts an entity into the table. When inserting an entity into a table, you must specify values for the PartitionKey and RowKey system properties. Together, these properties form the primary key and must be unique within the table. Both the PartitionKey and RowKey values may be up to 64 KB in size. If you are using an integer value as a key, you should convert the integer to a fixed-width string, because they are canonically sorted. For example, you should convert the value 1 to 0000001 to ensure proper sorting.
+    short-summary: Insert an entity into a table.
     parameters:
         - name: --table-name -t
           type: string
-          short-summary: 'The name of the table to insert the entity into.'
+          short-summary: The name of the table to insert the entity into.
         - name: --entity -e
           type: list
-          short-summary: 'A space-separated list of key=value pairs. Must contain a PartitionKey and a RowKey.'
+          short-summary: A space-separated list of key=value pairs. Must contain a PartitionKey and a RowKey.
+          long-summary: The PartitionKey and RowKey must be unique within the table, and may be up to 64Kb in size. If using an integer value as a key,
```

Comment:
Why is this long summary provided? It seems unnecessarily verbose and technical.

Reply:
Where there were comments of this type left by developers, I tended to leave them in since I assumed they were useful information to consumers of the API. In this particular case it seems like the lack of storage sorting on integers is important enough to call out in a description. Although I have no idea if it's actually true.

Reply:
The `The PartitionKey and RowKey must be unique within the table` feels like it should be part of the command's long-summary as it is not specific to the `--entity` param.

The size info should not be included for CLI.  In general, the CLI is a scenario focused tool and we want to focus on simplicity of our scenarios over specificity to function.  In this case, any user attempting to use a >64kb key combo would not find joy in the CLI experience...